### DISCUSSION TITLE
This knock-off PS3 controler just doesnt want to connect via USB

### DISCUSSION THREAD

**Discussion by Butter-On** [nefarius/DsHidMini#215] [Q&A] [answered]
I'm using some PS3 controller that my uncle gave me, and it doesn't work via USB. Is there something to help this?, the guide said about problems with Bluetooth not USB.
Also, it somehow doesn't appear in Device Manager but I installed it as the guide said?

**Comment by Butter-On** [comment]
Oh wow, it also doesn't appear in the registry?, is there any other software I need to install like VisualStudio C++?

**Comment by nefarius** [comment]
Well, are you sure it works? Do you have a PS3 as a reference to test? A different USB cable? Does it even light up when you press the PS button? Does it make the Windows connection sound?

**Comment by Butter-On** [comment, marked as answer]
Well, are you sure it works? - Yes Steam and built-in Windows calibration recognizes it.
Do you have a PS3 as a reference to test? - Unfortunately no.
A different USB cable? - The USB wire is un-pluggable from the controller.
Does it even light up when you press the PS button? - If you meant the 1,2,3,4 and the 1 lights up I guess?
Does it make the Windows connection sound? - Yes and it's recognized in devices.

**Reply by nefarius** [reply]
Well, it's hit and miss with 3rd party devices anyway. They either work, or not. You can try to dump a few months worth of research into why, but I'm done with that for good 😉

**Reply by Butter-On** [reply]
Well i got it working through steam, but yeah these controllers are hit or miss or miss sometimes
